sched/task/task_setup.c

```diff
@@ -121,17 +122,18 @@ static int nxtask_assign_pid(FAR struct tcb_s *tcb)
 #endif
           tcb->pid = next_pid;
 
-          sched_unlock();
-          return OK;
+          ret = OK;
+          goto errout;
         }
     }
 
+errout:
   /* If we get here, then the g_pidhash[] table is completely full.
```

Comment:
This comment is not correct any more since this is now _the_ return path.  Also maybe the label name should change.

Reply:
Sorry, I don't catch your comment. @masayuki2009 could you create a PR to fix the comment.

Reply:
@btashton 
> This comment is not correct any more since this is now the return path. Also maybe the label name should change.

Thanks for the comment

@xiaoxiang781216 
>Sorry, I don't catch your comment. @masayuki2009 could you create a PR to fix the comment.

Yes, I will create a new PR.